conans/test/utils/cpp_test_files.py

```diff
@@ -94,6 +94,22 @@ def imports(self):
     {other_calls}
 }}
 """
+
+body_c = r"""#include "hello{name}.h"
```

Comment:
I would recommend, adding the:

```python
def config(self):
     del self.settings.compiler.libcxx
```
for the C project case, as it will be the real use case